pkg/sql/schema_change_migrations_test.go

```diff
@@ -905,3 +905,75 @@ func TestGCJobCreated(t *testing.T) {
 		[][]string{{"1"}},
 	)
 }
+
+// TestMissingMutation tests that a malformed table descriptor with a
+// MutationJob but no Mutation for the given job causes the job to fail with an
+// error.
+func TestMissingMutation(t *testing.T) {
```

Comment:
nit: may be useful to include that this is a regression test for #48786 to have a concrete example to point to.